src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -116,6 +117,7 @@ const appendViewLinks = (
             actions: [{ name: skipStep?.name, params: rest }],
           });
         }),
+        step: transaction.nextStep!.name,
```

Comment:
I do not believe nextStep will be present here, this is a terminal scenario, so more likely than not there is no nextStep in transaction.